EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1679,7 +1679,7 @@ QUEST_LV_0300_20160310_001678	Lucien Winterspoon asked you to plant the herb see
 QUEST_LV_0300_20160310_001679	Talk with the villager, Nella
 QUEST_LV_0300_20160310_001680	It seems that Nella who've come back to the village has some request to you. Talk with her.
 QUEST_LV_0300_20160310_001681	Defeat the monsters at Nahas Forest
-QUEST_LV_0300_20160310_001682	The villager, Nella feels uneasy because she doesn��t know when she will get attacked again. She asked you to defeat the monsters that are wandering around the village.
+QUEST_LV_0300_20160310_001682	The villager, Nella feels uneasy because she doesn't know when she will get attacked again. She asked you to defeat the monsters that are wandering around the village.
```

Comment:
`Villager Nella feels uneasy because she doesn't know when she will get attacked again. She asked you to defeat the monsters that are wandering around the village.`